src/Angular.js

```diff
@@ -798,6 +802,21 @@ function arrayRemove(array, value) {
  *   and on `destination`) will be ignored.
  * </div>
  *
+ * <div class="alert alert-warning">
+ *   `angular.copy` does not check if destination and source are of the same type. It's the
+ *   developers responsibility to make sure they are compatible.
+ * </div>
+ *
+ * @knownIssue
+ * This is a list of (known) object types / features that are not handled correctly by this function:
```

Comment:
I'd make it more explicit that it is a non-exhaustive list. (I know that is what you mean with `(known)`; I'd just make it more explicit.)